shuup/admin/modules/orders/views/list.py

```diff
@@ -60,7 +60,7 @@ class OrderListView(PicotableListView):
         Column("payment_status", _(u"Payment Status"), filter_config=ChoicesFilter(choices=PaymentStatus.choices)),
         Column("shipping_status", _(u"Shipping Status"), filter_config=ChoicesFilter(choices=ShippingStatus.choices)),
         Column(
-            "taxful_total_price", _(u"Total"), sort_field="taxful_total_price_value",
+            "taxful_total_price_value", _(u"Total"), sort_field="taxful_total_price_value",
```

Comment:
Thanks for report, it's usually when `get_object_abstract` has hard coded keys, for example `item["taxful_total_price"]` will fix that